projects/ng2-ui-auth/src/public_api.ts

```diff
@@ -28,8 +29,9 @@ export {
   BrowserStorageService,
   AuthService,
   ConfigService,
+  IPartialConfigOptions,
```

Comment:
This was not exposed, and we use it. Not sure how it worked before; maybe the released version didn't match master?